lib/value_test.jsonnet

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env -S jx -J //github.com/yugui/jsonnetunit/raw/master
+
+local test = import 'jsonnetunit/test.libsonnet';
+local value = import 'value.jsonnet';
+
+{ name: 'value.jsonnet test' } +
+test.suite({
+  local getFixture = { a: 1, b: null, c: [1, 2, 3], d: { x: 'x', y: { z: 'z' } } },
+  testGetNoPath: {
+    actual: value.get(getFixture, []),
+    expect: getFixture,
+  },
+  testGetStringPath: {
+    actual: value.get(getFixture, 'a'),
+    expect: 1,
+  },
+  testGetPathOneLevel: {
+    actual: value.get(getFixture, 'a'),
+    expect: 1,
+  },
+  testGetArray: {
+    actual: value.get(getFixture.c, 2),
+    expect: 3,
+  },
+  testGetObjDefault: {
+    actual: value.get(getFixture, 'none', 'default'),
```

Comment:
maybe something different to `'none'` that doesn't make you wonder if it could be keyword?

Reply:
Done